source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/notifications/index.tsx

```diff
@@ -104,11 +110,30 @@ class Notifications extends React.Component<IProps> {
   }
 
   toggle = (e) => {
+
+    if (this.state.visible) {
+      this.markNotificationsToSeen();
+    }
+
     this.setState({
       visible: !this.state.visible
     });
   }
 
+  sortNotifications = (notifications) => {
```

Comment:
the first part of this sort doesn't do anything, and I know it was just extracted from render before, and I meant to say something earlier, but what is happening here, is that that the notifications are being sorted by time, and then resorted by isViewed, which does not consider the sort by time. 

once the second sort runs, it's possible that notifications will not be in time-order. 

ya don't need to fix it with this PR, since you just moved it from render, but just be aware of it in the future :)

Reply:
Will fix it later